src/clib/lib/job_queue/torque_driver.cpp

```diff
@@ -745,3 +750,18 @@ int torque_driver_get_submit_sleep(const torque_driver_type *driver) {
 FILE *torque_driver_get_debug_stream(const torque_driver_type *driver) {
     return driver->debug_stream;
 }
+
+ERT_CLIB_SUBMODULE("torque_driver", m) {
```

Comment:
Merge with `_torque_driver` which is already there?

Reply:
You are right